Source/JavaScriptCore/dfg/DFGCriticalEdgeBreakingPhase.cpp

```diff
@@ -76,6 +78,7 @@ class CriticalEdgeBreakingPhase : public Phase {
                     pad->predecessors.append(block);
                     (*successor)->replacePredecessor(block, pad);
                     successorPads.set(*successor, pad);
+                    m_graph.m_newJumpPads.append(pad);
                 } else
                     pad = iter->value;
 
```

Comment:
And then, in the end of this phase,
```
bool changed = m_insertionSet.execute();
if (changed && m_graph.m_form == ThreadedCPS)
    fixupJumpPadAvailability(newJumpPads);
return changed;
```

Reply:
The outputs of `DFGLoopUnrollingPhase.cpp` are `ThreadedCPS` (no unrolling) and `LoadStore` (loop unrolled). The reason is that `BlockInsertionSet::execute()` -> `m_graph.dethread()` which does `m_form = LoadStore;`. I think we can pass a flag via graph after loop unrolling phase in order to trigger `fixupJumpPadAvailability` in  `DFGCriticalEdgeBreakingPhase.cpp`.